test/test-pages/mozilla-1/expected.html

```diff
@@ -7,14 +7,7 @@
                 <div class="animation-wrapper" id="flexible-top-animation"><img src="http://mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/animations/flexible-top-fallback.c960365ba781.png" class="fallback" alt=""/></div>
             </div>
         </section>
-        <section id="designed" class="ga-section" data-ga-label="Designed to be redesigned">
-            <div class="container">
-                <div id="designed-copy">
-                    <h2>Designed to <br class="wide-br"/>be redesigned</h2>
-                    <p>Get fast and easy access to the features you use most in the new menu. Open the “Customize” panel to add, move or remove any button you want. Keep your favorite features — add-ons, private browsing, Sync and more — one quick click away.</p><img class="js " src="" data-processed="false" data-src="//mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/designed-redesigned.fbd3ee9402e6.png" data-high-res="true" data-high-res-src="//mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/designed-redesigned-high-res.6efd60766484.png" alt="" id="designed-mobile"/></div>
-                <div class="animation-wrapper" id="flexible-bottom-animation"><img src="http://mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/animations/flexible-bottom-fallback.cafd48a3d0a4.png" class="fallback" alt=""/></div>
-            </div>
```

Comment:
Honestly, I cannot understand why adding [`childNode.textContent.trim().length > 0`][change] condition cause it remove above `<div class="container">` node? Do you have any idea, or it is acceptable we remove it?

[change]: https://github.com/mozilla/readability/pull/337/files#diff-06d8d22df421dacde90a2268083424abR739